examples/02_meta-analyses/12_plot_ibma_workflow.py

```diff
@@ -0,0 +1,107 @@
+"""
+
+.. _ibma_workflow:
+
+====================================================
+Run an image-based meta-analysis (IBMA) workflow
+====================================================
+
+NiMARE provides a plethora of tools for performing meta-analyses on neuroimaging data.
+Sometimes it's difficult to know where to start, especially if you're new to meta-analysis.
+This tutorial will walk you through using a IBMA workflow function which puts together
+the fundamental steps of a IBMA meta-analysis.
+"""
+import os
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from nilearn.plotting import plot_stat_map
+
+from nimare.extract import download_nidm_pain
+
+###############################################################################
+# Download data
+# -----------------------------------------------------------------------------
+
+dset_dir = download_nidm_pain()
+
+###############################################################################
+# Load Dataset
+# -----------------------------------------------------------------------------
+import nibabel as nib
+from nilearn.image import resample_to_img
+
+from nimare.dataset import Dataset
+from nimare.transforms import ImageTransformer
+from nimare.utils import get_resource_path
+
+dset_file = os.path.join(get_resource_path(), "nidm_pain_dset.json")
+dset = Dataset(dset_file)
+dset.update_path(dset_dir)
+
+# Calculate missing images
+xformer = ImageTransformer(target=["varcope", "z"])
```

Comment:
it would be nice if we tried to handle the transformations for the user based on the algorithm they selected.

Reply:
that's a great idea!